modules/admin_manual/pages/release_notes.adoc

```diff
@@ -44,16 +44,22 @@ Version 10.2.1 corrects this behavior by dropping `share keys` for master key en
 
 === Solved Issues
 
-* **Fixed reshare privilege escalation issue** +
-ownCloud Server 10.2.0 introduced an issue that allows users to increase sharing permissions beyond their own permissions in a reshare scenario: When user A shares a folder "_Project_" with user B, granting only read and share permission, then the Sharing API allowed user B to reshare a subfolder of "_Project_" with user C granting full permissions. 
+* **Fixed reshare permission issue** +
+ownCloud Server 10.2.0 introduced an issue that allows users to increase sharing permissions beyond their own permissions in a reshare scenario: When user A shares a folder "_Project_" with user B, granting only read and share permission, then the Sharing API allowed user B to reshare a subfolder of "_Project_" with user C granting full permissions or to create a public link on the shared folder, respectively. 
 This undesired behavior is fixed with 10.2.1.
+* **Fixed issue with Sharing API and enforced public link expiration dates**
+An issue in the Sharing API caused the ownCloud clients to prevent users from creating public links when the option "Enforce expiration date" for public links is in use. This is now solved.
 * **Fixed known issue with user avatar paths** +
 Version 10.2.0 {release-notes-known-issues-url}[accidentally changed the location of user avatars] making them unavailable and storing uploaded avatar images in the wrong location.
 10.2.1 restores the earlier behavior and provides a repair step to move back the avatar images uploaded with 10.2.0 to the right location. 
 As it is not necessary nor possible to run `occ upgrade` when upgrading from 10.2.0 to this patch release, **you need to run `occ maintenance:repair -s 'OC\Repair\MoveAvatarIntoSubFolder'` manually to trigger the repair step**.
 * **Fixed https://doc.owncloud.com/server/admin_manual/release_notes.html#known-issues[known issue] with "Password changed" HTML emails rendered in plain text**
 * **Fixed use of invalid token on password reset** +
 Password reset links sent to a user were invalid, if the user attempted to login using their e-mail address and an invalid password prior to filling out and submitting the Reset Password form.
+* **Fixed issue when removing a user from a group**
+Remowing a user from a group using the user management UI resulted in an error that prevented the page to be refreshed to show the changes. This has been taken care of.
```

Comment:
This sentence doesn't quite read correctly. I suggest changing it to:: Remowing a user from a group using the user management UI resulted in an error that prevented the page to be refreshed from showing the changes. This has been corrected.

Reply:
IMO the problem was that the changes did not show in real-time, and you had to refresh the page to see that the action really worked?
If that is so, then:

Removing a user from a group using the user management UI resulted in an error that required the page to be refreshed to show the changes. This has been corrected.

Reply:
yes, thx both.